lib/node_modules/@stdlib/stats/base/smediansorted/benchmark/c/benchmark.length.c

```diff
@@ -24,7 +24,7 @@
 #include <sys/time.h>
 
 #define NAME "smediansorted"
-#define ITERATIONS 10000000
+#define ITERATIONS 1000000
```

Comment:
```suggestion
#define ITERATIONS 10000000
```

There was a reason this was higher in the original. Namely, to improve measurement accuracy. Ideally, a benchmark takes about `1` second to complete. We try to tune the number of iterations accordingly.